docs/reference/query-dsl/has-child-query.asciidoc

```diff
@@ -9,7 +9,7 @@ an example:
 --------------------------------------------------
 {
     "has_child" : {
-        "type" : "blog_tag",
+        "child_type" : "blog_tag",
```

Comment:
I've always used `type` here, never `child_type` (and `type` is what is documented)